extensions/podman/src/macos-checks.ts

```diff
@@ -67,23 +66,28 @@ export class MacPodmanInstallCheck extends BaseCheck {
   title = 'Podman Installation';
   async execute(): Promise<extensionApi.CheckResult> {
     // we need to check if brew is installed to avoid unexpected error
-    const brewResult = await runCliCommand('which', ['brew']);
-    if (brewResult.exitCode !== 0) {
-      // brew is not installed so do not check if podman has been installed with brew
-      return this.createSuccessfulResult();
+    try {
+      await extensionApi.process.exec('which', ['brew']);
+    } catch (err) {
+      if ((err as extensionApi.RunError) && err.exitCode !== 0) {
```

Comment:
why do we check the exit code ? if it's catch block it means we have not exitCode being 0

Reply:
updated